lfucg-exactions/client/js/components/AccountExisting.js

```diff
@@ -27,9 +26,26 @@ class AccountExisting extends React.Component {
         const {
             currentUser,
             accounts,
-            onAccountQuery,
+            plats,
+            lots,
         } = this.props;
 
+        const platsList = plats && plats.length > 0 &&
```

Comment:
@JMStewart00 Just FYI the `plats.length > 0` is unnecessary. Mapping over an empty array will return an empty array